src/frontend/komponenter/Fagsak/Vedtak/VedtakBegrunnelserTabell/useVedtakBegrunnelseMultiselect.ts

```diff
@@ -98,6 +99,20 @@ const useVedtakBegrunnelseMultiselect = (
         }
     };
 
+    const skalLeggeTilAndreBegrunnelse = (
```

Comment:
what do you think about
```suggestion
    const skalLeggeTilOpplysningspliktBegrunnelse = (
```

Reply:
Marcus said that we will need more "andre begrunnelser" in the future so maybe it is better to keep it like this